src/app_errors/mod.rs

```diff
@@ -134,16 +135,19 @@ pub enum AppErrorKind {
     #[fail(display = "Email account sent complaint")]
     BounceComplaintError {
         address: String,
+        bounced_at: DateTime<Utc>,
```

Comment:
Is it okay if we add a `#[serde(rename = "notificationType")]` attribute to these, so that the serialized property name matches what's expected by the auth server?

https://github.com/mozilla/fxa-auth-server/blob/3d5fd4c1bcd495b767c77dfac83a0d4a93994519/lib/senders/email_service.js#L57

Reply:
Whoops, copy-paste fail, I meant `bouncedAt` not `notificationType` of course. 😊

Reply:
Also, is `DateTime` serialised as a string or as an epoch timestamp? The auth server tests look like they're expecting the latter, just wanted to check we're doing the same thing here:

https://github.com/mozilla/fxa-auth-server/blob/3d5fd4c1bcd495b767c77dfac83a0d4a93994519/test/local/senders/index.js#L293

Reply:
> Is it okay if we add a #[serde(rename = "notificationType")] attribute to these, so that the serialized property name matches what's expected by the auth server?

I think that is not necessary, since we are not implementing Serialize at all for this enum. The way we get the data we need is through the `additional_fields` function.